src/module/meme-object.js

```diff
@@ -1,11 +1,32 @@
 let memeList = [];
 // === set local storage === //
-const setLocal = () => localStorage.setItem('memelist', JSON.stringify(memeList));
+const setLocal = () => {
+  const data = localStorage.setItem('memelist', JSON.stringify(memeList));
+  if (data !== null) {
+    return data;
+  }
+  return [];
+};
 // ==================== //
 // === get local storage === //
 const getLocal = () => JSON.parse(localStorage.getItem('memelist'));
```

Comment:
- [ ] It seems that you still have that problem with the local storage. Because I am getting the same error. Kindly fix it:
![Screenshot from 2022-03-29 20-30-39](https://user-images.githubusercontent.com/62705947/160681396-01c182ad-968f-433a-b343-61ef5e488691.png)

Reply:
yes right. 
I have by mistake add the check to the set local not get local.